Docker Tools: Add SSL support section (#4476)
microsoft/vscode-docs: docs/containers/debug-netcore.md

1. Optionally, set a breakpoint.
1. Start debugging! (`kb(workbench.action.debug.start)`)

## Running and debugging with SSL support

> If you are using SSL (using the HTTPS protocol), you will need to make a few changes to your configuration.

Reviewer: ```suggestion
To enable SSL (using the HTTPS protocol), you will need to make a few changes to your configuration.
```
